src/commands/shell.mjs

```diff
@@ -72,21 +83,16 @@ async function buildCustomEval(argv) {
 
       let res;
       try {
-        res = await performQuery(client, cmd, undefined, {
+        res = await runQueryFromString(cmd, {
           ...argv,
```

Comment:
any reason to destructure, then rest argv instead of passing it as-is?

Reply:
No. This will be cleaned up in a future revision.